app/src/main/java/com/infomaniak/mail/data/cache/mailboxContent/MessageController.kt

```diff
@@ -84,11 +83,11 @@ object MessageController {
         return ThreadController.getThread(threadUid)?.messages?.query()?.sort(Message::date.name, Sort.ASCENDING)
     }
 
-    fun getMessage(uid: String, realm: TypedRealm): Message? {
+    fun getMessage(uid: String, realm: TypedRealm? = null): Message? {
```

Comment:
instead of null you can directly use `RealmDatabase.mailboxContent()`

Reply:
I did it for all Controllers.